src/pint/models/stand_alone_psr_binaries/DDK_model.py

```diff
@@ -46,6 +49,12 @@ def __init__(self, t=None, input_params=None):
         del self.param_default_value["SINI"]
         self.set_param_values()
 
+    def __setattr__(self, name, value):
```

Comment:
Usually we do validation at the stage where we're evaluating the model. Other parameter errors and warnings are emitted there. Is it necessary to intercept `__setattr__`? This can make errors very difficult to track down.